src/transaction.rs

```diff
@@ -1661,7 +1660,7 @@ mod tests {
 
         // Output with pushes that are e.g. OP_1 are nulldata but not pegouts
         let output: TxOut = hex_deserialize!("\
-            0a2d3634393536d9a2d0aaba3823f442fb24363831fdfd0101010101010101010\
+            0a319c0000000000d3d3d3d3d3d3d3d3d3d3d3d3fdfdfd0101010101010101010\
```

Comment:
Do you remember why this changed?

Reply:
The test was incorrect. The asset is not a correct curve point